owncloud/test/test.py

```diff
@@ -782,6 +782,37 @@ def test_delete_attribute(self, attr1, value1):
         self.assertIsNone(self.client.get_attribute(self.app_name, attr1))
         self.assertEquals(self.client.get_attribute(self.app_name), [])
 
+class TestUserAndGroupActions(unittest.TestCase):
+
+    def setUp(self):
+        self.client = owncloud.Client(Config['owncloud_url'], single_session = Config['single_session'])
```

Comment:
I've implemented a unittest.SkipTest in case if client.get_apps() returns "False" for 'provisioning_api' or raises a ResponseError
https://github.com/individual-it/pyocclient/blob/a66412c20f6c87d3e48722685e0a84ed45f35096/owncloud/test/test.py#L799